src/services/repo/useRepoComponentsSelect.spec.tsx

```diff
@@ -34,7 +34,7 @@ const dataReturned = {
   owner: {
     repository: {
       __typename: 'Repository',
-      components: [
+      componentsYaml: [
```

Comment:
should add the `id` in the mock as well right?

Reply:
yes! good catch